### ISSUE TITLE
led screen doesn't read when led updated

### ISSUE THREAD

**Issue by tballmsft** [microsoft/microcode#122]
When you change the status of an LED in the icon editor, the new status is not read out.

**Comment by oldukhno** [comment]
issue had been fixed with https://github.com/microsoft/microcode/commit/43f84a56cb68423d17d5af2a5912ea2d6d276532
